engine/commands/engine_uninstall_cmd.cc

```diff
@@ -4,18 +4,8 @@
 
 namespace commands {
 
-EngineUninstallCmd::EngineUninstallCmd(std::string engine)
-    : engine_{std::move(engine)} {}
-
-void EngineUninstallCmd::Exec() const {
-  CTL_INF("Uninstall engine " + engine_);
-  auto engine_service = EngineService();
-
-  try {
-    engine_service.UninstallEngine(engine_);
-    CLI_LOG("Engine " << engine_ << " uninstalled successfully!")
-  } catch (const std::exception& e) {
-    CLI_LOG("Failed to uninstall engine " << engine_ << ": " << e.what());
-  }
+void EngineUninstallCmd::Exec(const std::string& engine) {
+  engine_service_.UninstallEngine(engine);
```

Comment:
Should we return a boolean value for `engine_service_.UninstallEngine`? Seems like we always delete engine successfully.

Reply:
yeah, I think would like to using exception for it. if no exception, then it will be a success. However, I found

Reply:
<img width="912" alt="Screenshot 2024-09-17 at 11 07 36" src="https://github.com/user-attachments/assets/c72d420e-753b-44b1-af9b-70088d23837e">

If the uninstallation is failed, it will be handled via exception here.